src/pages/home/report/ReportActionItem.tsx

```diff
@@ -631,103 +655,104 @@ function ReportActionItem(props) {
+                policyID={report.policyID}
             />
         );
     }
-    if (props.action.actionName === CONST.REPORT.ACTIONS.TYPE.RENAMED) {
-        return <RenameAction action={props.action} />;
+    if (action.actionName === CONST.REPORT.ACTIONS.TYPE.RENAMED) {
+        return <RenameAction action={action} />;
     }
-    if (props.action.actionName === CONST.REPORT.ACTIONS.TYPE.CHRONOSOOOLIST) {
+    if (action.actionName === CONST.REPORT.ACTIONS.TYPE.CHRONOSOOOLIST) {
         return (
             <ChronosOOOListActions
-                action={props.action}
-                reportID={props.report.reportID}
+                action={action}
+                reportID={report.reportID}
             />
         );
     }
 
     // For the `pay` IOU action on non-send money flow, we don't want to render anything if `isWaitingOnBankAccount` is true
     // Otherwise, we will see two system messages informing the payee needs to add a bank account or wallet
-    if (
-        props.action.actionName === CONST.REPORT.ACTIONS.TYPE.IOU &&
-        lodashGet(props.report, 'isWaitingOnBankAccount', false) &&
-        originalMessage &&
-        originalMessage.type === CONST.IOU.REPORT_ACTION_TYPE.PAY &&
-        !isSendingMoney
-    ) {
+    if (action.actionName === CONST.REPORT.ACTIONS.TYPE.IOU && !!report?.isWaitingOnBankAccount && action.originalMessage.type === CONST.IOU.REPORT_ACTION_TYPE.PAY && !isSendingMoney) {
         return null;
     }
 
-    const hasErrors = !_.isEmpty(props.action.errors);
-    const whisperedToAccountIDs = props.action.whisperedToAccountIDs || [];
+    const hasErrors = !isEmptyObject(action.errors);
+    const whisperedToAccountIDs = action.whisperedToAccountIDs ?? [];
     const isWhisper = whisperedToAccountIDs.length > 0;
     const isMultipleParticipant = whisperedToAccountIDs.length > 1;
     const isWhisperOnlyVisibleByUser = isWhisper && ReportUtils.isCurrentUserTheOnlyParticipant(whisperedToAccountIDs);
-    const whisperedToPersonalDetails = isWhisper ? _.filter(personalDetails, (details) => _.includes(whisperedToAccountIDs, details.accountID)) : [];
+    const whisperedToPersonalDetails = isWhisper
+        ? (Object.values(personalDetails ?? {}).filter((details) => whisperedToAccountIDs.includes(details?.accountID ?? -1)) as OnyxTypes.PersonalDetails[])
+        : [];
     const displayNamesWithTooltips = isWhisper ? ReportUtils.getDisplayNamesWithTooltips(whisperedToPersonalDetails, isMultipleParticipant) : [];
     return (
         <PressableWithSecondaryInteraction
             ref={popoverAnchorRef}
-            style={[props.action.pendingAction === CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE ? styles.pointerEventsNone : styles.pointerEventsAuto]}
-            onPressIn={() => props.isSmallScreenWidth && DeviceCapabilities.canUseTouchScreen() && ControlSelection.block()}
+            // TODO - Remove this once we have a better way to handle this
+            onPress={() => {}}
```

Comment:
What's the issue here?

Reply:
Issue here is that `onPress` prop is required on `PressableWithSecondaryInteraction` component but earlier it wasn't provided so I am not sure if I should just make it optional on the component pass just nothing or something else